app/controllers/administrator_invitations_controller.rb

```diff
@@ -0,0 +1,17 @@
+class AdministratorInvitationsController < ApplicationController
+  before_action :authenticate_administrator!
+  before_action :set_user
+
+  # PUT /admin_invitations/:id
+  # PATCH /admin_invitations/:id
+  def update
+    @user.invite!
+    flash[:success] = t('devise.invitations.send_instructions', email: @user.email)
+    redirect_to user_path(@user)
+  end
+
+  private
+  def set_user
+    @user = User.find(params[:id])
```

Comment:
In case you are wondering why I didn't user find_by, I want this to throw `ActiveRecord::RecordNotFound` exception if this user is not found, we should prolly have a proper/global (controller/concern) `ExceptionHandler` (if you don't have one already) so we don't have to check/handle on each controller/action.